editors/vscode/src/client.ts

```diff
@@ -32,7 +31,6 @@ let inset : WebviewEditorInset;
 export function activate(context: ExtensionContext) {
 
 	//create the server
-
 	let serverOptions = {
 		command: '/Users/houdamouzoun/.opam/4.05.0/bin/lp-lsp',
```

Comment:
hardcoded user-specific path

Reply:
Fixed in https://github.com/Deducteam/lambdapi/pull/240/commits/65b60d11c15cfdb5b737e94f5cf6b3bd14060d71